### ISSUE TITLE
Invalid sprayer section setup

### ISSUE THREAD

**Issue by jhomer** [RivalAUT/FS19_SprayerSectionControl#2]
Hello, wondering if you have any suggestions for me?
I recently modded the in-game Hardi Rubicon to work with your mod.
I set it up having A LOT of sections. Everything works as it should in-game without section control, but when section control is enabled I first get these errors while the game is loading for every section past the 10th section (not the first 10):
    Error: Index not found: testAreaWidth11
Warning: Invalid sprayer section setup 'vehicle.sprayerSectionControl.sections.section(10)' in 'C:/Users/James.000/Desktop/Georgetown_NY/Fs19_Rubicon/rubicon9000.xml'
Then if I use the sprayer when the game is loaded I get these errors that just don't stop:
   Error: Running LUA method 'update'.
C:/Users/James.000/Desktop/Georgetown_NY/FS19_SprayerSectionControl/SprayerSectionControl.lua:159: attempt to index a nil value

I looked over the xml and i3d mappings quite a few times and don't see any issues. Seems odd it's only showing an error after the first 10 sections.
Any thoughts you might have for me? 
Thanks.

**Comment by RivalAUT** [comment]
Thats strange. The indices of testAreaWidth11, 12 etc are correct in the i3dMappings?

**Comment by jhomer** [comment]
Problem solved! My apologies for wasting your time! Thanks for mentioning the i3d mappings, I went and had another look. 
Just a stupid spelling error on my part, missing an "a".
In the i3d mappings I had testAreWidth instead of testAreaWidth.
I misspelled it in the actual i3d itself and that is where I copied it from for the xml.
I made each nozzle for the sprayer individually controlled so I ended up with 90 sections with 92 spray effects.
Should have been 92 sections for 92 spray effects but a miscalculation and I made each workArea 0.01 too big, so ended with 90 work areas.
Took a surprising amount of time to do it so I was always looking for quicker ways to do things and ended up finding tools and shortcuts in notepad++ that I didn't even know existed.
